dev-docs/services/ci/README.md

```diff
@@ -0,0 +1,244 @@
+# The CI Service in Hail Batch
+
+Hail Batch includes a CI service which has three functional purposes:
```

Comment:
This is boarderline pedantry, but I don't think of "Hail Batch" being anything other than the batch service and the python client library. the batch service + ci etc are generally referred to as the "Services".
```suggestion
# The CI Service

The CI service has three functional purposes:
```